plugins/woocommerce/tests/php/includes/settings/class-wc-settings-general-test.php

```diff
@@ -1,4 +1,4 @@
-<?php
+wj<?php
```

Comment:
typo?

Reply:
It is 😃 Thank you for catching it!

Reply:
@psealock Fixed in https://github.com/woocommerce/woocommerce/pull/46140/commits/e25d8d0a48d3c08f9c3a5be5bd18ddee728a551a